keyboards/macro1/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+ENCODER_ENABLE = yes
+
```

Comment:
```suggestion
```

Reply:
change made

Reply:
This is still outstanding.

Reply:
Can you please help me understand what you are after? I removed this and then you requested to add it back in.

Reply:
No, you removed the wrong one. Look at the file itself, this is in the wrong place.

Reply:
I see now, sorry, fixed now.